templates/search/_filters_radius_citizen_projects.html.twig

```diff
@@ -0,0 +1,9 @@
+<div>
+    <select name="{{ constant('PARAMETER_RADIUS', search) }}">
+        {% for option in constant('RADII', search) %}
```

Comment:
`0` is allowed here?

Reply:
Yes. For show all Citizen Projects, the value is `0`.